ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -104,6 +106,9 @@ function WorkloadCvesOverviewPage() {
                                         setDefaultWatchedImageName(imageName);
                                         watchedImagesModalToggle.openSelect();
                                     }}
+                                    onWatchedImageLabelClick={() => {
```

Comment:
@dvail @mansursyed @vjwilson  I was thinking about what happens when we click on the `Watched image` label. So we seem to open up the modal based on the changes in this PR but I had some thoughts on that. Ideally, I would expect clicking on the label would do something related to that watched image. For example, open the modal and filter the table to show me that watched image or somehow highlight it. Otherwise, clicking on it is basically just like clicking on the `Manage watched images` button. We aren't doing any filtering, so it makes me think that we should leave it unclickable. What do you all think?

Reply:
I agree, I think it does feel a little weird to open the modal without any context specifically to that image. We could build in the ability to search the list (client side) and have this button pre-filter the list, but I think that inflates the scope a bit for something we mostly just want to port over.

If we go the route of making the button not clickable, does it make sense to change the label style to grey?
![image](https://github.com/stackrox/stackrox/assets/1292638/b8bcd6eb-878c-4ce9-8142-2637f798c007)

Reply:
Since there are multiple comments about making this unclickable, I'm updating the PR and reverting the `onClick` handler for this label in order to get the stack merged. We can revisit in a follow up if needed.